specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json

```diff
@@ -284,6 +284,46 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/privateLinkScopeOperationStatuses/{asyncOperationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get specific operation status": {
+            "$ref": "./examples/privateLinkScopeOperationStatuses.json"
+          }
+        },
+        "tags": [
+          "asyncOperations"
+        ],
+        "operationId": "GetAsyncOperationsStatus",
+        "description": "Get the status of an azure asynchronous operation associated with a private link scope operation.",
+        "parameters": [
+          {
+            "name": "asyncOperationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The operation Id."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Operation status.",
+            "schema": {
+              "$ref": "#/definitions/OperationStatus"
+            }
```

Comment:
Should have a default and refer to ErrorDetails under specification\common-type\v1\type.json. You can document the expected error code in the description section.

Reply:
didn't find it anywhere. thanks

Reply:
See example in [hybridcompute](https://github.com/Azure/azure-rest-api-specs/blob/34ea001f2c8b49411863b571d3c66de8d80ca30b/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2019-12-12/HybridCompute.json#L82)

Reply:
Hi Arolshan, will you also change this? Looks you miss this one